spec/system/hiring_staff_can_sign_in_with_dfe_spec.rb

```diff
@@ -146,27 +148,25 @@
       stub_sign_in_with_multiple_organisations
     end
 
-    context "when user preferences have been set" do
-      it_behaves_like "a successful sign in"
+    it_behaves_like "a successful sign in"
```

Comment:
We do this twice now at the same indentation?

Reply:
One is for trust and one is for local authority